topcoffea/modules/eft_helper.py

```diff
@@ -6,7 +6,7 @@
 from numba.typed import List
 import math
 
-@numba.njit
+#@numba.njit
```

Comment:
Does this not work with Numba? How does he performance change without using the JIT?

Reply:
Ah thanks. I disabled it because a previous attempt didn't work, but it should work with the current implementation,